src/control-center/src/main.cpp

```diff
@@ -1,32 +1,55 @@
 #include <iostream>
 #include <ControlCenter.h>
 #include <Serializer.h>
+#include <Logger.h>
+#include <ConfigReader.h>
 #include <csignal>
 #include <chrono>
 #include <thread>
 
 using namespace std;
 
-Serializer s;
-ControlCenter cc(s);
-bool quit = false;
+namespace {
+    static const int8_t BROADCAST_SENSORS_INTERVAL = 3;
```

Comment:
Jeśli trzymasz interwał czasowy w zwykłej zmiennej całkowitoliczbowej, to dodawaj w nazwie jednostkę - ułatwia czytanie i modyfikację (nie trzeba patrzeć do kodu).

Dodatkowo, gdy już definiujesz zmienną w przestrzeni nienazwanej, to nie musisz dopisywać `static` - ma to takie samo znaczenie (ograniczenie definicji do danej jednostki kompilacji).